nodes/object_operators/ScGetParent.py

```diff
@@ -0,0 +1,25 @@
+import bpy
+
+from bpy.types import Node
+from .._base.node_base import ScNode
+from ...helper import focus_on_object
+
+class ScGetParent(Node, ScNode):
+    bl_idname = "ScGetParent"
+    bl_label = "Get Parent"
+    
+    def init(self, context):
+        super().init(context)
+        self.inputs.new("ScNodeSocketObject", "Object")
+        self.outputs.new("ScNodeSocketObject", "Parent")
+    
+    def error_condition(self):
+        return (
+            self.inputs["Object"].default_value == None
+        )
+    
+    def pre_execute(self):
+        focus_on_object(self.inputs["Object"].default_value)
```

Comment:
Same here...
No need to focus on the object

Reply:
Fixed.